Source/Hurl.Settings/Views/BrowsersPage.xaml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Page
+    x:Class="Hurl.Settings.Views.BrowsersPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:ctlabs="using:CommunityToolkit.WinUI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Hurl.Settings.Views"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:uc="using:Hurl.Settings.Controls"
+    xmlns:vm="using:Hurl.Settings.ViewModels"
+    mc:Ignorable="d">
+    <Page.DataContext>
+        <vm:BrowsersPageViewModel x:Name="ViewModel" />
+    </Page.DataContext>
+
+    <StackPanel MaxWidth="800" Spacing="16">
+        <StackPanel>
+            <TextBlock
+                Margin="2,0,0,4"
```

Comment:
When sizes are this small, there's often a better solution. Could be as simple as removing this.